src/ResolveUserDecorator.php

```diff
@@ -67,8 +59,8 @@ public function setAnnotationReader($reader)
      */
     public function __call($method, $params)
     {
-        if ($this->auth->check()) {
-            call_user_func([$this->wrapped, $this->userSetterMethod], $this->auth->user());
+        if ($this->getAuth()->guard()->check()) {
+            call_user_func([$this->wrapped, $this->userSetterMethod], $this->getAuth());
```

Comment:
This changes behaviour no? We used to pass the logged in user here. Now it's passing the auth manager.

Reply:
Yes, it does.  That is a typo.  Good call.